src/Command/Run.php

```diff
@@ -26,12 +26,12 @@ protected function configure()
         $this
             ->setName('run')
             ->addArgument('directory', InputArgument::REQUIRED, 'Path to the test directory')
-            ->addOption('host', null, InputOption::VALUE_OPTIONAL, 'Base host to use', null)
+            ->addOption('host', null, InputOption::VALUE_REQUIRED, 'Base host to use', null)
             ->addOption('allow-self-signed-certificate', null, InputOption::VALUE_NONE, 'Allow self signed ssl certificate')
-            ->addOption('dns', null, InputOption::VALUE_OPTIONAL, 'DNS Ip to use', '8.8.8.8')
+            ->addOption('dns', null, InputOption::VALUE_REQUIRED, 'DNS Ip to use', '8.8.8.8')
```

Comment:
I think the DNS option is now unused anyway and can be dropped.

Reply:
Good catch. see https://github.com/jolicode/asynit/pull/17